src/EnergyPlus/DaylightingManager.cc

```diff
@@ -5017,10 +5111,10 @@ namespace DaylightingManager {
 		for ( ShelfNum = 1; ShelfNum <= NumOfShelf; ++ShelfNum ) {
 			SurfNum = Shelf( ShelfNum ).Window;
 			//    IF (SurfNum > 0) THEN
```

Comment:
Yes and I scanned through the rest of that file and got rid of some other commented out fortran.